pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -216,9 +228,23 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 		o.CurrentReports = append(o.CurrentReports, report)
 		allExecutionIDs = append(allExecutionIDs, weid.WorkflowExecutionId)
 
+		if seenWorkflows[weid.WorkflowId] >= (r.config.F + 1) {
+			// Update the last seen round for this outcome. But this should only happen if the workflow is seen by F+1 nodes
+			outcome.LastSeenAt = outctx.SeqNr
+		}
+
 		o.Outcomes[weid.WorkflowId] = outcome
 	}
 
+	// We need to prune outcomes from previous workflows that are no longer relevant.
+	for workflowID, outcome := range o.Outcomes {
+		// TODO: 3,600 is the amount of rounds we allow as threshold. This should be configurable.
+		if outctx.SeqNr-outcome.LastSeenAt > 3600 {
```

Comment:
Put 3600 into a const or, better yet, just add this config somewhere.

Reply:
Any examples of config setup that I could use here, or just set it up as a param when constructing the struct?

Reply:
It probably can sit in the onchain config that we extract in newReportingPlugin - BTW I see there is a TODO there that we might need to address seprately. For now you can use a const there and later we will replace it with a configured value, if present.